roles/keystone/tasks/main.yml

```diff
@@ -18,35 +18,70 @@
   file: dest=/etc/keystone state=directory
 
 - name: keystone log dir
-  file: dest=/var/log/keystone state=directory mode=0755 owner=keystone
-        group=keystone
-
-- name: install keystone service
-  upstart_service: name=keystone user=keystone cmd=/usr/local/bin/keystone-all
-                   config_dirs=/etc/keystone
+  file: dest=/var/log/keystone state=directory mode=0755 owner=www-data
+        group=www-data
```

Comment:
Do we want to run applications as separate users rather than all under www-data? This could allow a Horizon bug to compromise keystone.